src/test/java/mate/academy/service/impl/RoleServiceImplTest.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.service.impl;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import mate.academy.dao.RoleDao;
+import mate.academy.model.Role;
+import mate.academy.service.RoleService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class RoleServiceImplTest {
+    private static final Role ROLE = new Role(Role.RoleName.USER);
```

Comment:
Is it correct to make this role a constant? Let's move it's creation to the setUp method